kaspresso/src/main/kotlin/com/kaspersky/kaspresso/device/screenshots/screenshotmaker/DocLocScreenshotMaker.kt

```diff
@@ -0,0 +1,16 @@
+package com.kaspersky.kaspresso.device.screenshots.screenshotmaker
+
+import java.io.File
+
+class DocLocScreenshotMaker(
```

Comment:
the purpose of the class?

Reply:
Same as above

Reply:
please add code comments with description why we need this class

Reply:
done